4001-5000/LIT4975Dersan.xml

```diff
@@ -41,18 +42,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <textClass>
                 <keywords>
                     <term key="Homily"/>
+                    <term key="ChristianLiterature"/>
                 </keywords>
             </textClass>
             <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
         </profileDesc>
         <revisionDesc>
             <change who="DE" when="2018-06-13">Created entity</change>
+            <change when="2022-09-19" who="DR">Merged with deleted LIT1661Homily by suggestion of Marcin Krawczuk</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <div type="bibliography"/>
-         <!---->
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="LIT4975Dersan" passive="PRS3261Cyrilof"/>
+                    <relation name="betmas:formerlyAlsoListedAs" active="LIT4975Dersan" passive="LIT1661Homily"/>
```

Comment:
I would suggest adding a relation with PRS8854Simeon since we have more homilies on the Presentation in the database.